docs/cluster-configuration.md

```diff
@@ -1117,7 +1117,7 @@ With the `Custom` profile, the cipher list should be expressed according to Open
 
 On plain k8s, where APIServer CR is not available, the default value will be `Intermediate`.
 
-## Configure Applications Aware Quota (AAQ)
+## Configure Application Aware Quota (AAQ)
 To enable the AAQ feature, add the `applicationAwareConfig` field to the HyperConverged spec, with an empty object 
 or with custom settings (see below).
 
```

Comment:
Can you please change all the "ApplicationsResourceQuota" to "ApplicationAwareResourceQuota" (i see one in this file  line 1132)

Reply:
done